src/pages/download.astro

```diff
@@ -0,0 +1,109 @@
+---
+import Layout from '../layouts/Layout.astro';
+import { marked } from 'marked';
+
+const latestRelease = await (
+	await fetch(
+		'https://api.github.com/repos/RedstoneTools/redstonetools/releases/latest',
+	)
+).json();
+
+const versionNumber = latestRelease.tag_name;
+const releaseUrl = latestRelease.html_url;
+const jarDownload = latestRelease.assets[0].browser_download_url;
+const sourceDownload = latestRelease.zipball_url;
+const body = latestRelease.body;
+let content: string;
+if (body) {
+	content = marked.parse(body, { mangle: false, headerIds: false });
+}
+---
+
+<Layout currentPage='Download'>
+	<main>
+		<div class='container'>
+			<a href={jarDownload} class='option jar' target='_blank'>
+				<div>
+					<h3>Jar</h3>
+					<p>Dowload the Jar then drag and drop into your mods folder.</p>
+				</div>
+			</a>
+			<a href='curseforge://install?addonId=849609' class='option curseforge'>
+				<div>
+					<h3>Curseforge</h3>
+					<p>Open in curseforge launcher</p>
+				</div>
+			</a>
+			<a
+				href='https://modrinth.com/mod/redstone-tools'
+				class='option modrinth'
+				target='_blank'
+			>
+				<div>
+					<h3>Modrinth</h3>
+					<p>Open the modrinth page</p>
+				</div>
+			</a>
+			<a href={sourceDownload} class='option source' target='_blank'>
+				<div>
+					<h3>Build from source</h3>
+					<p>
+						Download the source code and build the jar for yourself (only do
+						this if you know what you're doing)
+					</p>
+				</div>
+			</a>
+		</div>
+		<h2><a href={releaseUrl} target='_blank'>{versionNumber}</a></h2>
+		<div class='changelog' set:html={content} />
+	</main>
+	<style lang='scss' slot='head'>
```

Comment:
should be in a separate file so it doesnt get regenerated with every request

Reply:
how would putting it in a separate file fix that?

Reply:
cause then you can compile the scss beforehand, its also more organized

Reply:
the scss is pre compiled, I find it easier to work on a page when the scss is in the same file plus it keeps the file structure clean

Reply:
if im looking for css i dont expect it to be in an html file, having a separate file for css does not make the file structure messy as long as you keep things separated and dont throw everything into a single folder

Reply:
This isnt a html file its an astro file which contains the content of the page, I see what you mean and normally I would agree but theres so little css specific to the page that I dont think it warrants its own file

Reply:
still think its nice to keep things clean from the very beginning so things dont get messy later on, guess it works though